src/dos/dos_locale_data.cpp

```diff
@@ -116,47 +118,65 @@ const std::vector<KeyboardLayoutInfoEntry> LocaleData::KeyboardLayoutInfo = {
 	},
 	{
 		{ "ux" }, "US (international, QWERTY)",
+		// The QWERTY layer is almost identical to the US layout
+		AutodetectionPriority::High,
 		850,
 		KeyboardScript::LatinQwerty,
 	},
 	{
 		{ "co" }, "US (Colemak)",
+		// Undoubtedly a very niche/exotic layout
+		AutodetectionPriority::Low,
 	 	437,
 	 	KeyboardScript::LatinColemak,
 	},
 	{
 		{ "dv" }, "US (Dvorak)",
+		// Undoubtedly a very niche/exotic layout
```

Comment:
You might wanna keep "undoubtedly" for the dramatic effect... or just drop it from everywhere 😄

Reply:
OK, changed.